Calendar/BookingHelpers/BookingStaffHelpers.ps1

```diff
@@ -0,0 +1,55 @@
+﻿function Get-MBPermissions {
+    param($identity)
+    # Get the Mailbox Permissions
+    $MBPermissions = Get-MailboxPermission -Identity $identity -ErrorAction SilentlyContinue
+    return $MBPermissions
+}
+
+function Get-MBRecipientPermissions {
+    param($identity)
+    # Get the Mailbox Recipient Permissions
+    $MBRecipientPermissions = Get-RecipientPermission -Identity $identity -ErrorAction SilentlyContinue
+    return $MBRecipientPermissions
+}
+
+
+function CheckMyBaseOptionsRBACRole($identity) {
+    $RBACRole = Get-ManagementRoleAssignment -RoleAssignee $identity -Role "MyBaseOptions" | Select-Object -Property RoleAssignee, Role
+    return $RBACRole
```

Comment:
Is there going to be validation on returned value?

Reply:
yes, on logic layer